app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee

```diff
@@ -8,6 +8,18 @@ $(document).on 'initialize', '#proxy', ->
   $("input.error").tipsy
     trigger: "focus"
 
+  # Warning for "catch-all" rules
+  setUpWarningForPattern = (pattern) ->
```

Comment:
we don't care about different (non catch all) duplicated patterns, right?

Reply:
That would be a nice feature although not the scope of this PR/issue IMO.

Reply:
Or maybe that's part of the verification in backend already?

Reply:
No idea, but would be good to bring that up somewhere.